packages/runtime/test/lib/testUtils.ts

```diff
@@ -171,6 +171,21 @@ export async function uploadOwnToken(transportServices: TransportServices): Prom
     return response.value;
 }
 
+export async function uploadPersonalizedOwnToken(transportServices: TransportServices, forIdentity: string): Promise<TokenDTO> {
```

Comment:
why not simplfy extend the function above with an optional forIdentity parameter?